go.mod

```diff
@@ -37,6 +37,12 @@ require (
 	gopkg.in/rethinkdb/rethinkdb-go.v6 v6.2.2
 )
 
+require (
+	github.com/andybalholm/brotli v1.0.4 // indirect
+	github.com/valyala/bytebufferpool v1.0.0 // indirect
+	github.com/valyala/fasthttp v1.37.1-0.20220607072126-8a320890c08d // indirect
+)
```

Comment:
Somehow this does not merge into the existing section?

Reply:
Manually merging them in the same order and running `go mod tidy` sorts them as expected. :+1: